tests/test_validation.py

```diff
@@ -689,67 +693,74 @@ def validate_hypotest(
             1e-6,
             "toybased",
         ),
-        ('1bin_lumi', {'init_pars': 2, 'par_bounds': 2}, 1.0, "q", 4e-6, "asymptotics"),
         (
-            '1bin_normsys',
+            '1bin_lumi_qtilde',
+            {'init_pars': 2, 'par_bounds': 2},
+            1.0,
+            "qtilde",
```

Comment:
Note to self: Even if the naming change can be reverted this change from `q` -> `qtilde` needs to stay.